rma_reason/models/rma_reason.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Raumschmiede GmbH
+# Copyright 2024 BCIM
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class RmaReason(models.Model):
+
+    _name = "rma.reason"
+    _description = "Rma Reason"
+
+    name = fields.Char(required=True, translate=True)
+    description = fields.Text()
```

Comment:
Why name is `translatable` but not the `description`? Both should be translatable.